src/main/java/mate/academy/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.bookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
1) I guess it **must be unchecked exception** (extends RuntimeException), because I don't see you have a handler logic for this in service or controller layer. Don't forget to **remove "throws"** in registration methods
2) According to the task, you have to **handle** the RegistrationException **using the GlobalExceptionHandler**